packages/kbn-search-connectors/types/native_connectors.ts

```diff
@@ -783,6 +803,26 @@ export const NATIVE_CONNECTOR_DEFINITIONS: Record<string, NativeConnector | unde
   google_cloud_storage: {
     features: {},
     configuration: {
+      buckets: {
+        default_value: null,
+        depends_on: [],
+        display: DisplayType.TEXTAREA,
+        label: i18n.translate(
+          'searchConnectors.nativeConnectors.googleCloudStorage.bucketNameLabel',
+          {
+            defaultMessage: 'List of buckets',
```

Comment:
This is also different from [here](https://github.com/elastic/connectors/blob/f70ebb9d8718a57cbfc7dd1a0e4be22667d0aefc/connectors/sources/google_cloud_storage.py#L177)